plugins/welcome/src/components/WelcomePage/ToggleFeatureFlagButton.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { FC } from 'react';
+import { Button } from '@material-ui/core';
+import { FeatureFlagState, featureFlagsApiRef, useApi } from '@backstage/core';
+
+const ToggleFeatureFlagButton: FC<{}> = () => {
```

Comment:
this would have been an opportunity to give it a `flagId` prop and pur it in core, right? :)

Reply:
Not sure I understand. My intention was mostly to demonstrate a way of how you'd use it in your codebase rather than a reusable Component

Reply:
Ah right. I meant that core (or rather the app, through some avatar menu) wants this exact type of toggle for its settings thingy anyway :)

Not sure if plugin code is going to do a lot of toggling (mostly reading), but showing it off like this is still great.